ush/global_cycle.sh

```diff
@@ -311,8 +307,6 @@ export ENDSCRIPT=${ENDSCRIPT:-" "}
 #  Other variables.
 export PGMOUT=${PGMOUT:-${pgmout:-'&1'}}
 export PGMERR=${PGMERR:-${pgmerr:-'&2'}}
```

Comment:
Should these also be changed to default to `out` and `err`?

Reply:
Done at 0ca672b.